platform/linux-generic/include/odp/api/plat/time_inlines.h

```diff
@@ -44,14 +44,42 @@ static inline odp_time_t _odp_time_cur(void)
 	return _odp_timespec_cur();
 }
 
+static inline uint64_t _odp_time_hw_to_ns(odp_time_t time)
+{
+	uint64_t nsec;
+	uint64_t freq_hz = _odp_time_glob.hw_freq_hz;
+	uint64_t count = time.count;
+	uint64_t sec = 0;
+
+	if (count >= freq_hz) {
+		sec   = count / freq_hz;
+		count = count - sec * freq_hz;
+	}
+
+	nsec = (1000000000ULL * count) / freq_hz;
```

Comment:
Fixed in v2. Defined internal GIGA_HZ in this file. Time API defines (e.g. SEC_IN_NS) is not visible in this file, since the API file is included after this "inlines.h" file.